xml/System.Numerics/IBitwiseOperators`3.xml

```diff
@@ -12,31 +12,25 @@
   <TypeParameters>
     <TypeParameter Name="TSelf">
       <Constraints>
-        <InterfaceName>System.IBitwiseOperators&lt;TSelf,TOther,TResult&gt;</InterfaceName>
+        <InterfaceName>System.Numerics.IBitwiseOperators&lt;TSelf,TOther,TResult&gt;</InterfaceName>
       </Constraints>
     </TypeParameter>
     <TypeParameter Name="TOther" />
     <TypeParameter Name="TResult" />
   </TypeParameters>
   <Interfaces />
-  <Attributes>
-    <Attribute>
-      <AttributeName Language="C#">[System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")]</AttributeName>
-      <AttributeName Language="F#">[&lt;System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")&gt;]</AttributeName>
-    </Attribute>
-  </Attributes>
   <Docs>
-    <typeparam name="TSelf">The type that implements this interface.</typeparam>
-    <typeparam name="TOther">The type that will is used in the operation with <typeparamref name="TSelf" />.</typeparam>
-    <typeparam name="TResult">The type that contains the result of <typeparamref name="TSelf" /> op <typeparamref name="TOther" />.</typeparam>
-    <summary>Defines a mechanism for performing bitwise operations over two values.</summary>
+    <typeparam name="TSelf">To be added.</typeparam>
```

Comment:
Comment loss. Looks like we'll have to manually port the doc comments for the renamed interfaces.